src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -4661,6 +4701,24 @@ subroutine atm_compute_dyn_tend_work(nCells, nEdges, nVertices, nVertLevels_dumm
          end do
       end do
 
+!  Rayleigh damping on u 
+
+      if(config_rayleigh_damp_u) then
+         do k=nVertlevels-config_number_rayleigh_damp_u_levels+1,nVertLevels
+            rayleigh_damp_coef(k) = (real(k - (nVertLevels-config_number_rayleigh_damp_u_levels))/ &
+                                     real(config_number_rayleigh_damp_u_levels))/                  &
+                                        (config_rayleigh_damp_u_timescale*86400.)
```

Comment:
Would it be better to specify the damping timescale in seconds in the namelist so we don't need to multiply by 86400 here? That would also have the benefit of keeping our namelist options in mks units as much as possible. We could also consider pre-computing the inverse timescale to avoid a division here.

Reply:
appended _days to the config timescale so the units are clear.  The loop setting the coefficient has been moved outside the edge loop (it didn't depend on the edge), and pre-computed the inverse.